roles/openshift_node/templates/multipath.conf.j2

```diff
@@ -1,19 +1,5 @@
 # LIO iSCSI
```

Comment:
Move these comments to the ansible as well.

Reply:
Sure

Reply:
Done